lisa/features/security_profile.py

```diff
@@ -1,38 +1,78 @@
 # Copyright (c) Microsoft Corporation.
 # Licensed under the MIT license.
-from dataclasses import dataclass
+from dataclasses import dataclass, field
+from enum import Enum
 from functools import partial
 from typing import Any, Type, cast
 
 from dataclasses_json import dataclass_json
 
-from lisa import schema
+from lisa import schema, search_space
 from lisa.feature import Feature
+from lisa.util import LisaException, constants, field_metadata
 
 FEATURE_NAME_SECURITY_PROFILE = "Security_Profile"
 
 
+class SecurityProfileType(str, Enum):
+    Standard = constants.SECURITY_PROFILE_NONE
```

Comment:
The standard doesn't need. If some vm sizes doesn't support any, the feature won't be here.